server/core/src/main/java/org/infinispan/server/core/AbstractProtocolServer.java

```diff
@@ -112,6 +118,21 @@ protected void startTransport() {
       }
    }
 
+   public void stopTransport() {
+      if (transport != null && transportStatus == ComponentStatus.RUNNING)
+         transport.stop();
```

Comment:
I think you meant to have brackets encapsulate the entire rest of the method?

Reply:
Repushed